qiskit_bot/api.py

```diff
@@ -149,7 +149,16 @@ def on_pull_event(data):
     if data['action'] == 'opened':
         repo_name = data['repository']['full_name']
         pr_number = data['pull_request']['number']
+        pr = REPOS[repo_name].gh_repo.get_pull(pr_number)
         if repo_name in REPOS:
+            if data['pull_request']['author_association'] != 'MEMBER':
```

Comment:
I feel like we should have a configuration flag around this behavior (either server side or repo side) so we can enable this on particular repositories. Since while I'm sure we want this on terra and some other repos others I don't think want to do this. For example I wouldn't want to enable this on retworkx since we don't distinguish between contributors like this (we don't have qiskit-bot enabled on retworkx yet, but it was something I was debating adding soon).